mentorship/gsoc/2022/projects/new-docs-site-engine/README.md

```diff
@@ -1,49 +1,352 @@
 
-#### Project scope & deliverables
+- Using [docusaurus-plugin-remote-content](https://github.com/rdilweb/docusaurus-plugin-remote-content)
 
-The user docs site is designed to assist end-users in using a product or service. The excellent user docs site is significant because it provides an avenue for users to learn how to use the software, its features, tips, and tricks, and resolve common problems encountered when using the software. It also reduces support costs and is part of the corporate identity of the product. The excellent user docs site is a sign of the healthiness of the product, and the developer team. Without a good user docs site, a user may not know how to do the above things effectively and efficiently. User docs site can play a pivotal role in ensuring a product's success because excellent communication is and will always be at the heart of any business or development. Fantastic documentation takes that communication and puts it in a manageable framework that everyone can access for success.
+Its Docusaurus plugin downloads content from remote sources. With this plugin, we can write the Markdown for our content somewhere else, and use them on your Docusaurus site, without copying and pasting. This is something I am working on currently and trying to combine with GitHub action too.
 
-#### Expected Deliverables
+> Since the `Adding Multiple repository docs support feature` task was taking a lot of time so I moved to other work and I thought I will pick this up at last because this work needs a lot of time and research and I was not sure how to implement this feature.
 
-- Support for the latest link pointing to the documentation for the last release.
-- Use versioned documentation from external sources, ideally from version tags and branches.
-- Move documentation sources to code repositories so that we could enable documentation as code and make documentation patches a part of the contribution process.
-- Automation that builds the site from multiple repositories. It includes scripting, GitHub Actions, and automated configuration management.
-- GitHub workflows for continuous deployment of the website to GitHub Pages, Netlify, Vercel, etc.
-- Implement previews for the website and support for staging changes (e.g. pre-release documentation). Developer and Contributor documentation for the new engine, in collaboration with the documentation contributors.
+**NOTE:** Multiple repository docs support feature is not yet implemented completely and it is still in progress. As of now `Docs Multi-instance` support is added which is a`@docusaurus/plugin-content-docs` plugin. Still, we want some custom GitHub Action or automation to pull docs from multiple repositories and organize them in a new documentation site engine.
```

Comment:
Since this is a critical feature that is not implemented, we need Github issues for the specific tasks that are required to get this implemented.  Each one should be targeted and specific and should include any details you have about the work that is required.  The issues should reflect the possibility that different people might work on different issues.  I recommend tagging them with something like "multi-repo support" so we can easily pull them all up for review.